<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rasterLayerStatisticsClass</class>
 <widget class="QWidget" name="rasterLayerStatisticsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>rasterLayerStatistics</string>
  </property>
  <widget class="QLineEdit" name="rasterFileInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>输入栅格：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openRasterFile">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="AoGIS.qrc">
     <normaloff>:/AoGIS/res/openFile.png</normaloff>:/AoGIS/res/openFile.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="csvFileOutput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>输出表格：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveCsvFile">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="AoGIS.qrc">
     <normaloff>:/AoGIS/res/saveFile.png</normaloff>:/AoGIS/res/saveFile.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="begin">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QComboBox" name="chooseBand">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AoGIS.qrc"/>
 </resources>
 <connections/>
</ui>
